cypress/support/index.d.ts

```diff
@@ -12,19 +12,22 @@ declare global {
       // Cypress specs so VSCode will recognize custom command types.
   
       // Cypress-axe methods that are used to create `checkAxe` custom command 
-      checkA11y(context?: ElementContext, config?: RunOptions, callback?: (violations: Result[]) => void): void;
+      checkA11y(context?: ElementContext, config?: RunOptions, callback?: (violations: Result[]) => void, skipFailures?: boolean): void;
       injectAxe(): void;
   
       /**
        * Convenience method to run the axe-core accessibility scanner without having to establish 
        * `cy.injectAxe()` in a `beforeEach` block. This method also reports axe violations in the 
        * console output for debugging.
        * 
+       * @param skipFailures Set to true to report failures to the console without Cypress failing the test.
        * @param context Any valid node or CSS selector. Defaults to the Cypress containing `<div>`.
-       * @param axeRunConfig Add or change rules in the `axe.run` config object
+       * @param axeConfig Add or change rules in the `axe.run` config object
        * @see https://www.deque.com/axe/core-documentation/api-documentation/#api-name-axerun
+       * @param callback Provide a custom callback function to handle the violations array from the Results object
+       * @see https://www.deque.com/axe/core-documentation/api-documentation/#results-object
        */
-      checkAxe(context?: ElementContext, axeConfig?: RunOptions, callback?: (violations: Result[]) => void): void;
+      checkAxe(skipFailures?: boolean, context?: ElementContext, axeConfig?: RunOptions, callback?: (violations: Result[]) => void): void;
```

Comment:
Once fns start getting 3-4+ args and some are optional, I strongly prefer they start using object syntax instead of an ordered list. Can we convert this accordingly?
```suggestion
      checkAxe(options: {
        skipFailures?: boolean;
        context?: ElementContext;
        axeConfig?: RunOptions;
        callback?: (violations: Result[]) => void;
      }): void;
```